apollo-api/src/main/java/com/apollographql/android/api/graphql/Field.java

```diff
@@ -148,7 +148,9 @@ private String orderIndependentKeyForVariableArgument(Map<String, Object> object
     Object variable = objectMap.get(VARIABLE_NAME_KEY);
     //noinspection SuspiciousMethodCalls
     Object resolvedVariable = variables.valueMap().get(variable);
-    if (resolvedVariable instanceof Map) {
```

Comment:
@sav007 the next line `null.toString()` gives NPE (the diff doesn't show that) 

An alternative could be to return `String.valueOf(resolvedVariable)` which would avoid the need for a null check if that's preferred